app/models/topic_embed.rb

```diff
@@ -58,7 +58,8 @@ def self.import(user, url, title, contents, category_id: nil, cook_method: nil)
           raw: absolutize_urls(url, contents),
           skip_validations: true,
           cook_method: cook_method,
-          category: category_id || eh.try(:category_id)
+          category: category_id || eh.try(:category_id),
+          tags: tags,
```

Comment:
If tagging is disabled, we should exclude the tags from `create_args`

Reply:
Handled in 8288985